css_tasks/task9/js/forms.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-len */
+/* eslint-disable no-undef */
+
+import {jumbotronForm, footerForm} from './app.js';
+
+
+const validateJumbotronForm = (e) => {
+  validateName(jumbotronForm, e);
+  validateEmail(jumbotronForm, e);
+  validatePassword(jumbotronForm, e);
+};
+const validateFooterForm = (e) => {
+  validateName(footerForm, e);
+  validateEmail(footerForm, e);
+  validateMessage(footerForm, e);
+};
+
+//  Validate Name Input
+
+const validateName = (formName, e) => {
+  const name = formName.elements.name;
+  const nameText = 'Only latin letters';
+  const nameValue = name.value;
+  const namePattern = /^[a-zA-Z]+$/g;
+  hideWarning(name);
+  if (!namePattern.test(nameValue)) {
+    showWarning(name, nameText);
+    e.preventDefault();
+  }
+};
+
+//  Validate Email Input
+
+const validateEmail = (formName, e) => {
+  const email = formName.elements.email;
+  const emailText = 'Provide valid email address';
+  const emailValue = email.value;
+  const emailPattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  hideWarning(email);
+  if (!emailPattern.test(emailValue)) {
+    showWarning(email, emailText);
+    e.preventDefault();
+  }
+};
+
+// Validate Password Input
+
+const validatePassword = (formName, e) => {
+  const password = formName.elements.password;
+  const passwordText =
```

Comment:
I would replace messages like this and regexp to another files with constants, it will make a bit easier to manage them and base code much cleaner

Reply:
Done